src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/RuleExporter.cs

```diff
@@ -214,9 +231,56 @@ private static class OptionalWorkloadRules
             ///     Represents the evaluation items representing optional workloads.
             /// </summary>
             [ExportRule(nameof(SuggestedWorkload), PropertyPageContexts.ProjectSubscriptionService)]
+            [ExportDesignTimeBuildTargets(nameof(SuggestedWorkload))]
```

Comment:
not sure about this one. It is not used in vs-green, but not sure it is always used in VS as well?